<launch>
  <node pkg="hector_scanmatcher" type="hector_mapper" name="hector_mapper_urg_low" ns="urg_low" launch-prefix="nice -n 16" output="screen">
    <param name="scan_topic" value="/laser0/scan" />
    <param name="base_frame" value="/base_stabilized" />
    <param name="use_tf_scan_transformation" value="true"/>
    <param name="use_tf_pose_start_estimate" value="true"/>

    <param name="map_pub_period" value="5.0"/>

    <param name="map_resolution" value="0.05"/>
    <param name="map_size" value="1024"/>
    <param name="map_start_x" value="0.5"/>
    <param name="map_start_y" value="0.5"/>


    <param name="laser_z_min_value" value="0.0"/>
    <param name="laser_z_max_value" value="0.7"/>

<!--    <remap from="map" to="map_low" />-->
  </node>

  <node pkg="hector_scanmatcher" type="hector_mapper" name="hector_mapper_urg_high" ns="urg_high" launch-prefix="nice -n 16" output="screen">
    <param name="scan_topic" value="/laser0/scan" />
    <param name="base_frame" value="/base_stabilized" />
    <param name="use_tf_scan_transformation" value="true"/>
    <param name="use_tf_pose_start_estimate" value="true"/>

    <param name="map_pub_period" value="5.0"/>

    <param name="map_resolution" value="0.05"/>
    <param name="map_size" value="1024"/>
    <param name="map_start_x" value="0.5"/>
    <param name="map_start_y" value="0.5"/>


    <param name="laser_z_min_value" value="1.3"/>
    <param name="laser_z_max_value" value="3.5"/>

<!--    <remap from="map" to="map_high" />-->
  </node>
</launch>
